src/MPDevice.cpp

```diff
@@ -1994,7 +1994,7 @@ bool MPDevice::addChildToDB(MPNode* parentNodePt, MPNode* childNodePt)
         else
         {
             /* If the login name is supposed to be before the one we're trying to insert */
-            if (tempNextChildNodePt->getLogin() > childNodePt->getLogin())
+            if (tempNextChildNodePt->getLogin().compare(childNodePt->getLogin()) > 0)
```

Comment:
I'm still not sure about that... the last time I tried it the result ended up being incorrect because of locales :/